src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/speed/modes/vulcan/SpeedVulcanGround286.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015-2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ *
+ *
+ */
+package net.ccbluex.liquidbounce.features.module.modules.movement.speed.modes.vulcan
+import net.ccbluex.liquidbounce.event.events.MovementInputEvent
+import net.ccbluex.liquidbounce.event.handler
+import net.ccbluex.liquidbounce.features.module.modules.movement.speed.modes.SpeedBHopBase
+import net.ccbluex.liquidbounce.utils.entity.moving
+import net.ccbluex.liquidbounce.utils.entity.strafe
+import net.ccbluex.liquidbounce.event.events.PacketEvent
+import net.ccbluex.liquidbounce.event.events.PlayerJumpEvent
+import net.minecraft.network.packet.c2s.play.PlayerMoveC2SPacket
+import net.minecraft.entity.effect.StatusEffects
+import net.minecraft.util.shape.VoxelShapes
+
+/**
+ * @anticheat Vulcan
+ * @anticheatVersion V2.8.6
+ * @testedOn anticheat-test.com, eu.loyisa.cn
+ * @note flags on specific blocks such as fences
+ */
+object SpeedVulcanGround286 : SpeedBHopBase("VulcanGround286") {
+    private inline val HasSidewaysMovement: Boolean

```

Comment:
variable name is written wrong should be "hasSidewaysMovement", also I am pretty sure there is already one called like this somewhere.